frontend/src/scenes/authentication/signup/verify-email/verifyEmailLogic.ts

```diff
@@ -48,7 +48,7 @@ export const verifyEmailLogic = kea<verifyEmailLogicType>([
             {
                 requestVerificationLink: async ({ uuid }: { uuid: string }) => {
                     try {
-                        await api.create(`api/users/${uuid}/request_email_verification/`, { uuid })
+                        await api.create(`api/users/request_email_verification/`, { uuid })
```

Comment:
this is breaking currently valid verification links, right? so everyone who created an account and did not verify yet won't be able to verify? Or am I confusing this?

Reply:
(this comment actually makes more sense in line 34 sry)

Reply:
Nope.